### PR TITLE
Jsc/poco

### PR DESCRIPTION
**Pull request by jcubbage**
This looks like a good first pass out our initial models. Added initializers for all the tables.

### CODE REVIEW

File: Keas.Core/Domain/Location.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Keas.Core.Domain
+{
+    public class Location
```
Comment (srkirkland) [inline Keas.Core/Domain/Location.cs]: @jcubbage i forget is a location supposed to be tied to a team or are they system-wide

Comment (jSylvestre) [reply Keas.Core/Domain/Location.cs]: I think we said they were tied to a team...

**Comment by jcubbage** [comment]
Updated key references as requested. Inverse property not working on TeamMembers to Assignments and Assets. Not sure what I'm messing up, but I'm done for today. :)

**Comment by srkirkland** [comment]
@jcubbage We'll have to tweak all this stuff anyway so for now I say 🚢

**Pull request merged**